### PR TITLE
Remove git version and onboard to Appveyor

### PR DESCRIPTION
**Pull request by CodeCyclone**
Changed to use Appveyor's version handling, locally its driven by Directory.Build.props.
Fixes to packaging targets to allow usage of dotnet pack --no-build.
Scripts added to disable strong name and update PowerShell manifest during packaging.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  file: 'src\Directory.Build.props'
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
+
+# Install
+install:
+  - ps: 'scripts\DisableStrongName.ps1'
```
Comment (nandemoii) [inline appveyor.yml]: It might be helpful to add comments here or in the script itself stating why this is necessary.

Comment (CodeCyclone) [reply appveyor.yml]: I will add it to the script

File: scripts/UpdatePSModuleManifest.ps1 [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+if(($psdFileContent -match $matchRegex) -and ($Matches['ModVersion'] -ne $Version)) {
+    Write-Output "Updating ModuleVersion to: $Version"
+    $psdFileContent = $psdFileContent -replace $matchRegex, $replaceMatch
+    $updatePsdFile = $true
+}
+else {
+    Write-Output "Module '$ModulePath' already contains ModuleVersion $($Matches['ModVersion'])"
+}
+
+# Remove Prelease tag
```
Comment (nandemoii) [inline scripts/UpdatePSModuleManifest.ps1]: Prelease -> Prerelease

Comment (CodeCyclone) [reply scripts/UpdatePSModuleManifest.ps1]: Good catch

**Pull request merged**